app/views/reports/setup.js.erb

```diff
@@ -21,7 +21,6 @@ $('#defined_reports_step_2').html("<%= raw escape_javascript(render(:partial =>
 $('#report_selection').hide();
 $('#defined_reports, #defined_reports_step_2').show();
 // $('#defined_reports_step_1').dialog('close');
-$("select").selectpicker()
```

Comment:
When using bootstrap, we use selectpicker js for all select boxes, for consistent styling/features. At least in fulfillment, @kyle-glick is this still correct for sparc?